packages/dashboard/src/components/pages/home/home.tsx

```diff
@@ -1,17 +1,193 @@
+/* eslint-disable react/display-name */
 import * as React from 'react';
-import { Text } from '@deriv/components';
+import { DesktopWrapper, Text, VerticalTab } from '@deriv/components';
+
 import TempButtons from 'Components/temp-buttons';
 
 const Home: React.FC = () => {
-    return (
-        <React.Fragment>
-            <div className='dw-home'>
-                <Text as='p' size='l' color='less-prominent' align='center'>
-                    Home
+    const list = [
+        {
+            default: true,
+            icon: 'IcUserOutline',
+            label: 'My Deriv',
```

Comment:
Labels should be `localize`d.

Reply:
Thanks, I planned to add it later since we don't know if these labels are finalized, but we can add now as well